bundles/org.openhab.binding.rotel/src/main/resources/OH-INF/config/config.xml

```diff
@@ -320,4 +320,43 @@
 		</parameter>
 	</config-description>
 
+	<config-description uri="thing-type:rotel:serial6">
+		<parameter name="newerUnit" type="boolean" required="false">
```

Comment:
> "newer" is relative, so there is a risk this name will not age well. As it seems to control the baud rate, perhaps it could be named directly for this purpose, e.g. highBaudRate?

Yes, good idea.

> Is it possible to read the serial number from the device?

No

Reply:
> You could also consider making it integer with options to set the baud rate directly to 19200 or 38400 (and having descriptions explaining the choices).
> 
> You would still be able to detect which volume command to use based on thing type, and it wouldn't be mixed with the baud rate.

I could have one unique model in the enum but that would then
 require to pass the baud rate (config) as overriding parameter in few classes.

Could be better, maybe.

Reply:
I was wrong, I have nothing difficult to change to pass the baud rate in other classes.
So I take the option to have `baudRate` as parameter.